src/features/events/components/eventsPageBanner/EventsPageBanner.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import { Row, Column } from '../../../../components/grid'
+import { checkBreakpoint } from '../../../../utilities/'
+import {
+  StyledBanner,
+  Title,
+  Subtitle,
+  ResponsiveImg,
+  ImgWrapper,
+} from './EventsPageBanner.styles'
+import { EventsPageBannerProps } from './EventsPageBanner.types'
+
+export const EventsPageBanner: React.FC<EventsPageBannerProps> = ({
+  title,
+  subtitle,
+  backgroundColor,
+  image,
+}) => {
+  console.log(image)
+  return (
+    <StyledBanner role="banner" backgroundColor={backgroundColor}>
+      <Row
+        alignItems="center"
+        flexDirection={'row-reverse'}
+        justifyContent={'space-between'}
+        mr={0}
+        flexWrap={{ default: 'wrap', md: 'nowrap' }}
+      >
+        <Column
+          pr={0}
+          py="0"
+          maxWidth="920px"
+          width={{ default: 1, md: 0.6666 }}
+          mb={{ default: '-20px', md: 0 }}
+        >
+          <ImgWrapper>
+            <ResponsiveImg
+              fixed={
+                !checkBreakpoint(480)
+                  ? image.mobile
+                  : !checkBreakpoint(768)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['magic numbers' are not allowed: 768](https://app.codacy.com/app/PrideInLondon/pride-london-web/pullRequest?prid=4694377)